htsget-search/src/htsget/from_storage.rs

```diff
@@ -21,9 +22,9 @@ where
   fn search(&self, query: Query) -> Result<Response> {
     match query.format {
       Some(Format::Bam) | None => BamSearch::new(&self.storage).search(query),
+      Some(Format::Cram) => CramSearch::new(&self.storage).search(query),
       Some(Format::Vcf) => VcfSearch::new(&self.storage).search(query),
-      Some(Format::Bcf) => BcfSearch::new(&self.storage).search(query),
-      Some(format) => Err(HtsGetError::unsupported_format(format)),
```

Comment:
I reckon it's worth to keep the unsupported format pattern here, there are others that might come this way and we've not implemented yet in the spec (BED, GFF, etc...)

Reply:
That pattern could never be reached because its an enum type. Perhaps we should introduce an "unsupported" enum option?

Reply:
I think we could do that or we would need to detect that error in the htsget-http crate (I've just encountered this "problem"). I feel like it would be more logical to handle it here, but could be either way

Reply:
+1 for the enum option Marko suggested, here seems to be the right place to put it.